docs/system-overview.md

```diff
@@ -0,0 +1,77 @@
+_This document is intended to read top-to-bottom. Do yourself a favour and read the whole thing without skipping ;)_
+
+# Formula Student Driverless Simulation: System overview
+
+FSDS is built around Unreal Engine 4 (the game engine) and the AirSim plugin. 
+The game engine does all the graphical rendering, collision simulation and car movement simulation. 
+A separate component - the simulator - will handle all controll the simulation, provide external interfaces and store what is happening.
+
+! There will be also something that provides a video feed for livestreaming the simulation. 
+  Since it is still unclear how this will work this is not included in this description.
+
+![System overview](images/system-overview.png)
+
+## Autonomous Systems.
+
+Every autonomous system run on their own seperated environment. 
+Idealy this would be separate virtual machines but also Docker containers could be used.
+The autonomous systems are expected to continuously run a ROS master.
```

Comment:
Has this been decided?

Reply:
The alternative would be that the autonomous systems are only running a ROS master when they are 'in use'. This would result in having the simulator somehow start the ros autonomous system within the team's environment. Since ROS is not running at that point, we would have to setup another communication channel to signal that system to start it's ros core. An extra communication channel is a lot of work to build, test and document for both us and the teams. That's why I think we should definitely go for having the autonomous systems run their ros masters continuously. How the simulator will deal with this is indeed an open question but I believe something we can figure out.

Reply:
let's start with this concept in our testing environment then!